--- /etc/dovecot/conf.d/10-logging.conf
+++ /etc/dovecot/conf.d/10-logging.conf
@@ -4,10 +4,10 @@
 
 # Log file to use for error messages. "syslog" logs to syslog,
 # /dev/stderr logs to stderr.
-#log_path = syslog
+log_path = /dev/stderr
 
 # Log file to use for informational messages. Defaults to log_path.
-#info_log_path = 
+info_log_path = /dev/stdout
 # Log file to use for debug messages. Defaults to info_log_path.
 #debug_log_path = 
 

