--- /etc/dovecot/conf.d/15-mailboxes.conf
+++ /etc/dovecot/conf.d/15-mailboxes.conf
@@ -46,12 +46,15 @@
 namespace inbox {
   # These mailboxes are widely used and could perhaps be created automatically:
   mailbox Drafts {
+    auto = subscribe
     special_use = \Drafts
   }
   mailbox Junk {
+    auto = subscribe
     special_use = \Junk
   }
   mailbox Trash {
+    auto = subscribe
     special_use = \Trash
   }

