--- /etc/dovecot/conf.d/90-sieve.conf
+++ /etc/dovecot/conf.d/90-sieve.conf
@@ -74,6 +74,7 @@
   #sieve_after =
   #sieve_after2 =
   #sieve_after2 = (etc...)
+  sieve_after = /etc/dovecot/sieve-after
 
   # Which Sieve language extensions are available to users. By default, all
   # supported extensions are available, except for deprecated extensions or
@@ -112,7 +112,7 @@
   # localpart and uses the first one encountered. The :user part is
   # left of the separator and the :detail part is right. This setting
   # is also used by Dovecot's LMTP service.
-  #recipient_delimiter = +
+  recipient_delimiter = -

   # The maximum size of a Sieve script. The compiler will refuse to compile any
   # script larger than this limit. If set to 0, no limit on the script size is
